### ISSUE TITLE
cinstall doesn't install PDB debug symbol files with MSVC

### ISSUE THREAD

**Issue by nirbheek** [lu-zero/cargo-c#279]
With MinGW, the debug information is embedded in the object files, but with MSVC it is split out into PDB files. These need to be installed alongside the object files to get symbols in the Visual Studio debugger. Some references to how meson handles this:

https://github.com/mesonbuild/meson/blob/a692395186b9a74817a584444418901da1d7306f/mesonbuild/backend/ninjabackend.py#L2427-L2474

https://github.com/mesonbuild/meson/blob/a692395186b9a74817a584444418901da1d7306f/mesonbuild/backend/ninjabackend.py#L3069-L3074

Compile args:

https://github.com/mesonbuild/meson/blob/a692395186b9a74817a584444418901da1d7306f/mesonbuild/compilers/mixins/visualstudio.py#L304-L308

https://github.com/mesonbuild/meson/blob/a692395186b9a74817a584444418901da1d7306f/mesonbuild/compilers/mixins/visualstudio.py#L410-L420

Linker args:

https://github.com/mesonbuild/meson/blob/a692395186b9a74817a584444418901da1d7306f/mesonbuild/linkers/linkers.py#L1225-L1230

tl;dr: if you install foo.dll you must also install foo.pdb next to it. Same with foo.exe. You can ignore static libraries.

**Comment by nirbheek** [comment]
I was looking at fixing this, but it looks like cargo work might be needed to get this working correctly, because there doesn't seem to be a 'corresponding PDB' for built targets, so that would need to be fixed first.

**Comment by lu-zero** [comment]
Open an issue on cargo and link this one so once it is solved we won't forget about it :)

**Comment by nirbheek** [comment]
Looks like I was wrong, cargo does build `foo.pdb` files for each library in the release directory.
